pkg/volume/rbd/rbd_util.go

```diff
@@ -137,9 +138,13 @@ func (util *RBDUtil) rbdLock(b rbdMounter, lock bool) error {
 		// for defencing, get the locker name, something like "client.1234"
 		cmd, err = b.plugin.execCommand("rbd",
 			append([]string{"lock", "list", b.Image, "--pool", b.Pool, "--id", b.Id, "-m", mon}, secret_opt...))
-		output = string(cmd)
-		glog.Infof("lock list output %q", output)
+		if cmd == nil {
```

Comment:
Hi,
If the rbd is not found, the output is:
rbdLock **cmd: ""**; **err: executable file not found in $PATH**

From my tests, the 'cmd' will alway be nil if there is no rbd cmd. 
I'm not sure whether the 'err' will be that string or not and I'll do some more tests.
Thanks.